README.md

```diff
@@ -378,7 +378,15 @@ const useStoreWithUndo = create<StoreState>()(
 
 ### Cool-off period
 
-`handleSet?: (handleSet: StoreApi<TState>['setState']) => StoreApi<TState>['setState']`
+```typescript
```

Comment:
I have some concerns that this type definition may spook new users reading the README, but it also seems like a good idea to be accurate here?